app/router.js

```diff
@@ -3,7 +3,8 @@ import RouterScroll from 'ember-router-scroll';
 import config from './config/environment';
 
 const Router = Ember.Router.extend(RouterScroll, {
-	location: config.locationType
+	location: config.locationType,
+	rootURL: config.rootURL
```

Comment:
I remember we had some issues when using `rootURL`, maybe they're no longer valid. Did you test it on sandbox? I think it was something about fingerprinting, so please check the file names if they're as expected.